ghu_web/ghu_main/views.py

```diff
@@ -15,7 +15,7 @@
 ]
 
 def home(request):
-    return render(request, 'ghu_main/home.html', {})
+    return render(request, 'ghu_main/about.html', {})
```

Comment:
hmmmmmmmmmmmmmmmmmmmmmmmm

i wanna tell you to add a url entry for `^$ → views.about` instead, but then we gotta change all the url references to `home` in templates, so that sucks

we could also change `views.home` to `views.about` in `urls.py`, but that's even worse imo. so i think this is the best approach we can take. if someone's wondering why home.html isn't doing anything, the first place they'd look (afaik) is the view, so they could fix this